README.md

```diff
@@ -92,6 +92,7 @@ Parameter        | Default value                                  | Description
 ---------------- | ---------------------------------------------- | -----------
 cookieName       | privacyCookieAccepted                          | Sets your own status cookie name
 cookieValidity   | 365                                            | Says how many days privacy banner should be hidden when user accepts policy?
+cookiePath       | null                                           | Specifies the cookie path
```

Comment:
We could add some context here:

` Specifies the cookie path (by default cookie will be available only for the current domain)`